content/en/news/2019/announcing-1.4/change-notes/index.md

```diff
@@ -16,8 +16,13 @@ weight: 10
 
 ## Telemetry
 
+- **Added** experimental in-proxy telemetry reporting to [Stackdriver](https://github.com/istio/istio/wiki/Mixerless-Stackdriver-HTTP-Telemetry).
+- **Added** experimental support for generating TCP service metrics [in-proxy](/docs/ops/telemetry/in-proxy-service-telemetry/).
+- **Improved** support for [in-proxy](/docs/ops/telemetry/in-proxy-service-telemetry/) Prometheus generation of HTTP service metrics (from experimental to alpha).
+- **Improved** telemetry collection for [blocked and passthrough external service traffic](/blog/2019/monitoring-external-service-traffic/).
 - **Added** the option to configure [stat patterns](/docs/reference/config/istio.mesh.v1alpha1/#MeshConfig) for Envoy stats.
 - **Added** the `inbound` and `outbound` prefixes to the Envoy HTTP stats to specify traffic direction.
+- **Improved** reporting of telemetry for traffic that goes through an egress gateway. 
```

Comment:
please remove trailing white spaces.  Ths trips up the linter.  The linter says the problem is on this line, but it may have an off by one error on whitespace problems.